src/bitcoin-cli.cpp

```diff
@@ -19,22 +19,27 @@ using namespace json_spirit;
 
 std::string HelpMessageCli()
 {
+#define SCREEN_SIZE 132
+#define INDENT 25
+#define MESSAGE_SIZE SCREEN_SIZE - INDENT
+#define OPT(option, message) option + FormatParagraph(message, MESSAGE_SIZE, INDENT) + "\n"
```

Comment:
Let's use type-safe constants and inline functions. I'd like to avoid C-style macros unless there is no other choice. e.g.

    static const int screenWidth = 132;
    ....
    static inline std::string opt(const std::string &option, const std::string &message)
    {
        return option + FormatParagraph(...) + "\n";
    }